src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write code here
+function sortList(list) {
```

Comment:
The function `sortList` is defined to sort a list of elements. Ensure that the `list` parameter is a valid DOM element with children to avoid runtime errors.